cmd/antrea-agent/options.go

```diff
@@ -738,3 +730,19 @@ func (o *Options) validateSecondaryNetworkConfig() error {
 
 	return nil
 }
+
+func (o *Options) valdateNodePortLocalConfig() error {
+	if !features.DefaultFeatureGate.Enabled(features.NodePortLocal) {
```

Comment:
I think you should define `o.enableNodePortLocal = *o.config.NodePortLocal.Enable && features.DefaultFeatureGate.Enabled(features.NodePortLocal)`, instead of mutating `o.config.NodePortLocal.Enable`. See #5401

Reply:
Done, Add a new variable `o.enableNodePortLocal`